src/lib/cpp/tests/test_fossdbmanagerclass.cc

```diff
@@ -183,6 +251,15 @@ CPPUNIT_TEST_SUITE(FoLibCPPTest);
     CPPUNIT_ASSERT(!result);
   }
 
+  /**
+   * Test to check if DbManager can create connection with scheduler connect
+   * \test
+   * -# Create a test sysconf dir with a VERSION file.
+   * -# Set the `argv`
+   * -# Call fo::DbManager::DbManager() with the `argv`.
+   * -# Execute a statement and check the result.
+   * \todo make this correctly
```

Comment:
let's have all the same issue for code todos (if that todo is serious still today)